tests/src/test/scala/ch/renku/acceptancetests/pages/ProjectPage.scala

```diff
@@ -189,7 +200,9 @@ class ProjectPage(projectDetails: ProjectDetails, userCredentials: UserCredentia
       }(waitUpTo(10 minutes), implicitly[source.Position])
 
     private def waitForImageToBeReady(implicit webDriver: WebDriver): Unit = {
-      find(cssSelector(".badge.badge-warning")) getOrElse fail("Image building info badges not found");
+      find(cssSelector(".badge.badge-warning")) getOrElse (findImageReadyBadge getOrElse fail(
```

Comment:
I think you should be able to simplify that with:
`find(cssSelector(".badge.badge-warning")) orElse findImageReadyBadge getOrElse fail("Image building info badges not found")`

Reply:
I initially tried that as well, but that results in a complication error: "value getOrElse is not a member of Object."